bin.src/createRefSpec.py

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python
+#createRefSpec.py '/Users/azuri/spectra/pfs/PFS' --id visit=4 filter='r' spectrograph=2 site='F' category='A' -C ../obs_pfs/config/pfs/createRefSpec.py --output "/Users/azuri/spectra/pfs/PFS" --clobber-config
```

Comment:
Added "# example:" before "#createRefSpec..." and removed the config override for the example, instead added the two config parameters as arguments